src/routing/RouterConfig.tsx

```diff
@@ -23,27 +23,34 @@ export const RouterConfig = () => {
   return (
     <BrowserRouter>
       <Routes>
-        <Route path={routes.index.path} element={<Index />} />
-        <Route path={routes.others.path} element={<NotFound />} />
-        <Route path={routes.mapShow.path} element={<MapShow />} />
-        <Route path={routes.serach.path} element={<Search />} />
-        <Route path={routes.qrRead.path} element={<QrRead />} />
-        <Route path={routes.stampRefer.path} element={<StampRefer />} />
-        <Route path={routes.login.path} element={<Login />} />
-        <Route path={routes.signup.path} element={<Signup />} />
-        <Route path={routes.signupConfirm.path} element={<SignupConfirm />} />
-        <Route path={routes.signupDone.path} element={<SignupDone />} />
-        <Route path={routes.passwordForget.path} element={<PasswordForget />} />
-        <Route path={routes.passwordReset.path} element={<PasswordReset />} />
-        <Route
-          path={routes.passwordResetDone.path}
-          element={<PasswordResetDone />}
-        />
-        <Route path={routes.passwordChange.path} element={<PasswordChange />} />
-        <Route
-          path={routes.passwordChangeDone.path}
-          element={<PasswordChangeDone />}
-        />
+        <Route path={routes.index.path} element={<Layout />}>
+          <Route path={routes.index.path} element={<Index />} />
+          <Route path={routes.others.path} element={<NotFound />} />
+          <Route path={routes.mapShow.path} element={<MapShow />} />
+          <Route path={routes.serach.path} element={<Search />} />
+          <Route path={routes.qrRead.path} element={<QrRead />} />
+          <Route path={routes.login.path} element={<Login />} />
```

Comment:
```suggestion
          <Route path={routes.signin.path} element={<Signin />} />
```

見つけてしまったので直して欲しいです。

`Signup`, `Login`が並んでいるのが問題だと思っています。
API上はsign*で表現するのでこちらに合わせて欲しいです